docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What maintainers should know before adopting Packit for downstream release automation 
+sidebar_position: 5
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages (Packit does not have yet a reverse dependencies check)
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent breaking changes in stable Fedora versions.
+
+:::
+
+:::warning Packit pull requests can spam other maintainers
+
+SPam -> ask more about the spam
+
+:::
+
+:::warning missing notifications
+
+Packit bot user creates pull requests, Koji builds, and Bodhi updates, making it the default notification recipient.
+
+To receive pull request comment notifications, configure notification settings accordingly.
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
+
+Track *Packit Service* activity through:
+- the pull request updates and 
+- the [Packit Dashboard](https://dashboard.packit.dev/).
+
+Packit Service responds with 👀 for every valid command in pull requests. Missing eyes indicate:
+- Incorrect command syntax
+- Packit Service outage
+
+Check service status at [https://status.packit.dev/](https://status.packit.dev/).
+
+## Recovery procedures
+
+### Service-based retriggering
+
+Each job configured in `packit.yml` can be retriggered via dist-git pull request comments. See the [complete list of commands](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#retriggering).
+
+### Manual CLI Operations
+
+During service outages, maintain workflow using Packit CLI:
+
+1. Install [Packit CLI](https://packit.dev/docs/cli)
+2. [Configure required tokens](https://packit.dev/docs/configuration#user-configuration-file)
+3. Verify setup:
+   - Follow [dist-git onboarding guide](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding)
+   - Validate config with `packit validate-config`
```

Comment:
If this is about service outages, I would expect users to already have working config.
What would be nice to document (or perhaps fix) are config differences between service and CLI. For example, a config without `downstream_package_name` works just fine with service but doesn't work with CLI.

Reply:
> If this is about service outages, I would expect users to already have working config. 

Yes, agreed, but I think this is however worth mentioning for a new user.

> What would be nice to document (or perhaps fix) are config differences between service and CLI. For example, a config without `downstream_package_name` works just fine with service but doesn't work with CLI.

I am not aware of any difference 😅 but sure that would be useful. And I would create a page just for them if we can collect all the differences. Not sure if they do fit in this, I would say more generic, guide.

Reply:
> I am not aware of any difference 😅 but sure that would be useful. And I would create a page just for them if we can collect all the differences. Not sure if they do fit in this, I would say more generic, guide.

I think ideally there should be no differences and we should fix those that exist, rather than documenting them. It surprised me too, I hit it a couple of times, people tend to leave `downstream_package_name` out as it works for them in service but then you try to reproduce their issue locally and it will traceback with a cryptic error.
There is one more thing I know of that works only in service, and that's job-level actions. But that's probably not fixable.